test/select.spec.js

```diff
@@ -636,6 +639,31 @@ describe('ui-select tests', function () {
     expect(el.find('.select2-search-choice-close').length).toEqual(1);
   });
 
+  it('should clear selection (with object as source)', function() {
```

Comment:
The main `should clear selection` test still uses `createUISelect`, but I didn't see a way to test the "object as source" syntax without compiling the template manually (I used the `should bind model correctly (with object as source)` test as a template). Any ideas how to make this cleaner?